### PR TITLE
Upgrade script v2 - fluentd.persistence migration

### PR DESCRIPTION
**Pull request by pmalek-sumo**
###### Description

Upgrade script v2 - fluentd.persistence migration

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: tests/upgrade_v2_script/static/fluentd_persistence_false.input.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        kubelet: /fluentd/buffer/metrics.kubelet
+        container: /fluentd/buffer/metrics.container
+        controller: /fluentd/buffer/metrics.controller
+        scheduler: /fluentd/buffer/metrics.scheduler
+        state: /fluentd/buffer/metrics.state
+        node: /fluentd/buffer/metrics.node
+        control-plane: /fluentd/buffer/metrics.control_plane
+        default: /fluentd/buffer/metrics.default
+      events: /fluentd/buffer/events
+      traces: /fluentd/buffer/traces
```
Comment (sumo-drosiek) [inline tests/upgrade_v2_script/static/fluentd_persistence_false.input.yaml:88]: Do all of this keys are required to check persistence?

Comment (pmalek-sumo) [reply tests/upgrade_v2_script/static/fluentd_persistence_false.input.yaml:88]: Nope but the more the better right :)? I can easily remove them

Comment (sumo-drosiek) [reply tests/upgrade_v2_script/static/fluentd_persistence_false.input.yaml:88]: It depends 😀 
I would drop them due to test redability

Comment (pmalek-sumo) [reply tests/upgrade_v2_script/static/fluentd_persistence_false.input.yaml]: done

Comment (pmalek-sumo) [reply tests/upgrade_v2_script/static/fluentd_persistence_false.input.yaml]: done v2

**Review by sumo-drosiek — APPROVED** [review]
Thanks

**Pull request merged**